### DISCUSSION TITLE
Does someone  use chroma.limits(data, 'e', 4);

### DISCUSSION THREAD

**Discussion by WohnungNDS** [open-sdg/open-sdg#1431] [Ideas]
**Is your feature request related to a problem? Please describe.**

Using chroma.limits(data, 'e', 4); to build  five classes for the map based on my data
Seen in the croma.brewer documentation [here](https://gka.github.io/chroma.js/#chroma-limits)

I'm not shure how to use it.

I tried this in my [site config file](https://github.com/WohnungNDS/IM_Site/blob/3ad22e8a8133507c311d8e8f7b00e3ad62936b03/_config.yml#L180)

`  colorRange: chroma.brewer.OrRd
  limits: chroma.limits(data, 'e', 5);`

It seems there is an effect on the map: only three colurs are shown, but in the legend there are still 9 coulors.

![image](https://user-images.githubusercontent.com/67795376/120180866-97605100-c20c-11eb-95b1-a2df57f0daab.png)

**Comment by LucyGwilliamAdmin** [comment]
@WohnungNDS I'm not sure if this is just being caused by the spread of your data, as the highest value for 2020 is Region Hannover at 1155330 with the next highest being 534049 for Lhst. Hannover, which is quite a big range compared to between the rest of the values, which may explain why only some colours are being used

**Comment by WohnungNDS** [comment]
@LucyGwilliamAdmin I haven't been into this topic for a couple of month, but I will get back in a few weeks.... Nevertheless this could be a reason.....
